api/app/controllers/api/v1/projects_controller.rb

```diff
@@ -37,7 +40,8 @@ def create
 
       def update
         @project = load_and_authorize_project
-        ::Updaters::Project.new(project_params).update(@project)
+        options = @project.draft? || project_params.dig(:draft) ? { touch: false } : {}
```

Comment:
Why is this line here? Again, this seems like the wrong place for this. Why not put it in the updater itself, if it's necessary?